domenetjenester/brevbestiller/pom.xml

```diff
@@ -99,6 +99,11 @@
             <artifactId>json</artifactId>
             <version>20201115</version>
         </dependency>
+        <dependency>
+            <groupId>com.googlecode.json-simple</groupId>
+            <artifactId>json-simple</artifactId>
+            <scope>test</scope>
+        </dependency>
 
```

Comment:
Kan du fjerne den over (20201115)?
Selv om testscope, så hadde vært greit å bruke Jackson (JsonNode i jackson-databind) og slippe disse ekstra bibliotekene.
Eksempel 
            JsonNode testdata = jsonMapper.readTree(testScenario.getString(scenario)) ;
            dto = jsonMapper.readValue(testdata.get("null").textValue(), BehandlingDto.class);

Reply:
Jeg tenker å merge for å få testen til å bygge og merget Espen sin PR slik at vi kjørere alle testene igjen. Men enig i at det ikke er nødvendig å bruke andre biblioteker om Jackson allerede har støtte for det. Jeg ser på å bytte etterpå.